tuf/ngclient/_internal/requests_fetcher.py

```diff
@@ -22,11 +22,11 @@
 
 # Classes
 class RequestsFetcher(FetcherInterface):
-    """A concrete implementation of FetcherInterface based on the Requests
+    """A concrete implementation of ``FetcherInterface`` based on the Requests
     library.
```

Comment:
Minor nit: According to our style guide, this should fit on one line. What about...
```suggestion
    """An implementation of ``FetcherInterface`` based on the requests library.
```
?